python/doc/bibliography.rst

```diff
@@ -167,6 +169,8 @@ Bibliography
     computational experiments.*
     Journal of Statistical Planning and Inference, 43 :381-402, 1995.
     `pdf <https://www.osti.gov/servlets/purl/10184343>`__
+.. [morokoff1995] Morokoff, W. J., & Caflisch, R. E. (1995). Quasi-monte carlo 
+    integration. Journal of computational physics, 122(2), 218-230.
```

Comment:
```suggestion
.. [morokoff1995] Morokoff, W. J., & Caflisch, R. E. (1995). *Quasi-Monte Carlo 
    integration.* Journal of computational physics, 122(2), 218-230.
```

Reply:
https://www.math.ucla.edu/~caflisch/Pubs/Pubs1995-1999/actaNumerica1998.pdf

Reply:
Thank you for the link. I added it.